minio/minio
# Check both given and normalized group DN on LDAP policy detach requests

## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description
Currently detaching a policy from an LDAP group only tries to detach from the given group DN. This changes that so both the given DN as well as the normalized DN are acted on, consistent with the behavior for users.

## Motivation and Context
Right now, it is possible to attach a policy to a group using a non-normalized version of the DN, but using the same non-normalized version of the DN on the detach fails.

## How to test this PR?
Attach a policy to a valid non-normalized DN, i.e.: `mc idp ldap policy attach local readwrite --group CN=TestGroup,CN=Users,DC=min,DC=io`

Now attempt to detach the same: `mc idp ldap policy detach local readwrite --group CN=TestGroup,CN=Users,DC=min,DC=io`

Previously this would error: `mc: <ERROR> Unable to make LDAP policy association. The specified policy change is already in effect. (Specified policy update has no net effect).`
Now it should work as expected

## Types of changes

## Checklist:
